app/src/main/java/com/example/android/uamp/viewmodels/MainActivityViewModel.kt

```diff
@@ -164,12 +167,23 @@ class MainActivityViewModel(
         }
     }
 
+    /**
+     * Sends a command via MusicServiceConnection to MusicService to toggle spatial audio
+     *
+     * @param enable current toggle state of spatial audio
+     */
+    suspend fun toggleSpatialization(enable: Boolean){
+        val bundle = Bundle()
+        bundle.putBoolean("EXTRAS_TOGGLE_SPATIALIZATION", enable)
```

Comment:
You should use the constant you defined in `MusicService` here, i.e. remove the quotation marks around `EXTRAS_TOGGLE_SPATIALIZATION`, similar to how you use `ACTION_TOGGLE_SPATIALIZATION` below :)